packages/coordinator/src/screens/bookManualDonationScreen/BookManualDonationScreen.tsx

```diff
@@ -30,12 +30,18 @@ export default function BookManualDonationScreen(
 ) {
   const [firstName, setFirstName] = useState("");
   const [firstNameError, setFirstNameError] =
-    useState<Validation.FirstNameValidation>();
+    useState<Validation.FirstNameValidation>(
```

Comment:
The `<type>` is now redundant (here and below)

Reply:
Without the type, 
it wouldn't be explicitly FirstNameValidation,
but PersonalDetailsValidation